packages/kmod-6.12-nvidia-r570/kmod-6.12-nvidia-r570.spec

```
@@ -362,6 +441,8 @@ popd
 %{_cross_unitdir}/load-tesla-kernel-modules.service
 %{_cross_unitdir}/copy-open-gpu-kernel-modules.service
 %{_cross_unitdir}/load-open-gpu-kernel-modules.service
+%{_cross_unitdir}/copy-grid-kernel-modules.service
+%{_cross_unitdir}/load-grid-kernel-modules.service
```

These should also be conditionalized, no?
```suggestion
%if "%{_cross_arch}" == "x86_64"
%{_cross_unitdir}/copy-grid-kernel-modules.service
%{_cross_unitdir}/load-grid-kernel-modules.service
%endif
```

---

I ran into an issue where if these aren't present, nvidia-persistend.service doesn't find the load and never starts on arm. I can confirm if this is really true.

---

I just confirmed that we can keep these conditionalized and ARM is happy. I'll update!